driver/InputFlags.cpp

```diff
@@ -55,6 +55,7 @@ void InputFlags::AddInputFlag(const std::string& _long_name,
 
 [[gnu::noreturn]] void InputFlags::Print() const
 {
+    printf("MIOpen Driver Input Flags: \n\n");
```

Comment:
@atamazov @technicalgrp89 : I'm curious if `printf` can be actually effective here.

Reply:
@junliume It was present, before I started working on this change. , so, I retained it. @atamazov

Reply:
Yes, this reverts change caught at https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1282#discussion_r763403764